src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const totalPrice = days * price;
+
+  if (days < 3) {
+    return totalPrice;
+  } else if (days < 7) {
+    return totalPrice - 20;
+  } else {
+    return totalPrice - 50;
+  }
```

Comment:
Last else is redundant here
```suggestion
  return totalPrice - 50;
```